src/components/SelectionList/CheckboxListItem.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import PressableWithFeedback from '../Pressable/PressableWithFeedback';
+import styles from '../../styles/styles';
+import Text from '../Text';
+import {checkboxListItemPropTypes} from './selectionListPropTypes';
+import Checkbox from '../Checkbox';
+import Avatar from '../Avatar';
+import OfflineWithFeedback from '../OfflineWithFeedback';
+import useLocalize from '../../hooks/useLocalize';
+
+const propTypes = {
+    /** The section list item */
+    item: PropTypes.shape(checkboxListItemPropTypes),
+
+    /** Whether this item is focused (for arrow key controls) */
+    isFocused: PropTypes.bool,
+
+    /** Callback to fire when the item is pressed */
+    onSelectRow: PropTypes.func,
+
+    /** Callback to fire when an error is dismissed */
+    onDismissError: PropTypes.func,
+};
+
+const defaultProps = {
+    item: {},
+    isFocused: false,
+    onSelectRow: () => {},
+    onDismissError: () => {},
+};
+
+function CheckboxListItem(props) {
```

Comment:
We should use prop destructuring now -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring